packages/neos-ui-redux-store/src/CR/Nodes/selectors.js

```diff
@@ -25,13 +36,25 @@ export const nodeByContextPath = state => contextPath => {
     return node;
 };
 
+export const makeGetDocumentNodes = nodeTypesRegistry => createSelector(
+    [
+        nodes
+    ],
+    nodesMap => {
+        const documentSubNodeTypes = nodeTypesRegistry.getSubTypesOf('TYPO3.Neos:Document');
+        const nodes = nodesMap.toJS();
+
+        return Object.values(nodes).filter(node => documentSubNodeTypes.includes(node.nodeType));
+    }
+);
+
 export const focusedNodePathSelector = createSelector(
     [
         focused,
-        currentDocumentNode
+        getCurrentContentCanvasContextPath
     ],
-    (focused, currentDocumentNode) => {
-        return focused || currentDocumentNode;
+    (focused, getCurrentContentCanvasContextPath) => {
```

Comment:
nitpick: just for readability, I would remove the `get` prefix for the parameter.